contentctl/objects/correlation_search.py

```diff
@@ -365,10 +400,6 @@ def _get_relevant_observables(observables: list[Observable]) -> list[Observable]
                 relevant.append(observable)
         return relevant
 
-    # TODO (PEX-484): ideally, we could handle this and the following init w/ a call to
```

Comment:
resolves PEX-484

Reply:
Agreed, I can see a call to this function added to model_post_init above